virttest/qemu_vm.py

```diff
@@ -1459,7 +1470,8 @@ def add_disable_modern(devices, dev, dev_type):
                     if params.get('machine_type').startswith("arm64-mmio"):
                         dev = QDevice('virtio-serial-device')
                     else:
-                        dev = QDevice('virtio-serial-pci', parent_bus=pci_bus)
+                        parent_bus = _get_pci_bus(port_params, "virtio_port")
```

Comment:
This virtio_port is not set anywhere in the config, therefor it defaults to the main `pcie.0`. Is this expected?